sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/XmlExternalEntityShouldNotBeParsed_XmlReader_BeforeNet452.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System.Xml;
```

Comment:
I wouldn't rename it BeforeNet452 because Net35 is before 4.5.2 as well

maybe use the same convention as for XmlDocument - Net4